instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py

```diff
@@ -91,13 +91,20 @@
     "system.memory.utilization": ["used", "free", "cached"],
     "system.swap.usage": ["used", "free"],
     "system.swap.utilization": ["used", "free"],
-    "system.disk.io": ["read", "write"],
-    "system.disk.operations": ["read", "write"],
-    "system.disk.time": ["read", "write"],
-    "system.network.dropped.packets": ["transmit", "receive"],
-    "system.network.packets": ["transmit", "receive"],
-    "system.network.errors": ["transmit", "receive"],
-    "system.network.io": ["transmit", "receive"],
+    "system.disk.io.read": None,
```

Comment:
Could you update the code example above as well in the doc string.

Reply:
I believe I have addressed this with the most recent commits